product_specifications/10_xromv_e.md

```diff
@@ -0,0 +1,43 @@
+
+## Problem
+
+There is no way to claim DOI and visualize which DOIs have been already claimed to someone's ORCID Record in DataCite Commons.
```

Comment:
What does 'claimed' in Commons mean currently?

Reply:
I think currently, in Commons, it holds a double meaning, because it also talk about "ORCID auto-update" DOIs as claimed. Which that might be a slightly miss-use of the term.